tinygrad/tensor.py

```diff
@@ -339,6 +339,11 @@ def chunk(self, num, dim):
     for i,k in enumerate(range(0, self.shape[dim], self.shape[dim]//num)):
       slice_params[i][dim] = (k, min(self.shape[dim], k+self.shape[dim]//num))
     return [self.slice(p) for p in slice_params]
+  
+  def interpolate(self, scale_factor, mode='nearest'):
```

Comment:
For PyTorch:
Currently temporal, spatial and volumetric sampling are supported, i.e. expected inputs are 3-D, 4-D or 5-D in shape.

That should be easy to copy, this also needs a test

Reply:
moved interpolate out of tensor.py, for full pytorch like implementation a seperate PR would be better